CONTRIBUTING.md

```diff
@@ -21,11 +21,10 @@ The developer process of releasing of a new version of StarcounterClientFiles `3
 4. Build the project using `build.bat` or VS. This triggers `msbuild`, which also triggers `npm run build`
 5. When you push to a feature branch of StarcounterClientFiles, TeamCity will run a integration test of this branch with KitchenSink and Blending. Make sure these tests are passing.
 6. Make a PR for someone to review the complete set of changes.
-7. Still in the feature branch, push a tag (3.x.y), and add release notes to GitHub release of StarcounterClientFiles.
-8. Still in the feature branch, publish a package of this version to App Warehouse (built with Starcounter 2.4). You may follow the steps to do this [here](https://github.com/Starcounter/CompanyTrack/blob/master/AppsTeam/Guidelines/releasing-to-warehouse.md). 
-9. Execute [package.bat](https://github.com/Starcounter/StarcounterClientFiles/blob/3.x/package.bat)
+7. Still in the feature branch, after PR approval, bump assembly version (in `AssemblyInfo.cs` under `Properties`), publish a package of this version to App Warehouse (built with Starcounter 2.4), and release to GitHub. You may follow the steps to do this [here](https://github.com/Starcounter/CompanyTrack/blob/master/AppsTeam/Guidelines/releasing-to-warehouse.md). 
+8. Execute [package.bat](https://github.com/Starcounter/StarcounterClientFiles/blob/3.x/package.bat)
```

Comment:
It's covered by 6. step of ["publish a package of this version to App Warehouse"](https://github.com/Starcounter/CompanyTrack/blob/master/AppsTeam/Guidelines/releasing-to-warehouse.md#releasing-apps-to-the-warehouse) => 6th sub-step of step 7. Do I really need to call it once again?

Reply:
It's not. The first one is to publish to Starcounter Warehouse. The second one is to publish a Nuget package. They're two different things.

Reply:
Good catch! Addressed.